src/pages/index.js

```diff
@@ -198,7 +199,7 @@ function handleDeleteCard(cardData) {
 
 function handleLikeIcon(cardData) {
   api
-    .setLike(cardData._id, cardData._isLiked)
+    .setLike(cardData.id, cardData.isLiked)
```

Comment:
`.setLike(cardData.id, cardData.isLiked)`

To make it work you need to change them in class `Card`. 

`this._id` should be `this.id` and `this._isLiked` should be `this.isLiked`